### PR TITLE
Add Website Link for Ferry Chandra

### LINKED ISSUE #16: FEY

### PR DESCRIPTION
**Pull request by ferry5245**
Add Website Link for Ferry Chandra

Fix #16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ Berikut merupakan orang-orang yang mengikuti academy ini :
 - [Wizman Rofiansyah](https://github.com/Rofiansyah) [WIZ]
   > Sebagai mentee
 - [Ferry Chandra](https://github.com/ferry5245) [FEY]
-  > Sebagai mentee
+  > Sebagai mentee  
+  > Website link -> https://ferry5245.github.io
```
Comment (fakhrip) [inline README.md]:
```suggestion
  > Website link -> https://ferry5245.github.io/
```

**Review by fakhrip — CHANGES_REQUESTED** [review]
You should add the remaining `/` character in the end of the link to match it with the other links

File: README.md (hunk 1 above)
Comment (fakhrip) [reply README.md]: You can fix this by `commit`-ing the changes and then push it to your forked repository and this PR will automatically match the last changes you made

**Comment by fakhrip** [comment]
Also dont forget to add a `Fix #16` to close the issue immediately after this PR merged

**Comment by IritaSee** [comment]
But the point of that hyperlink is to be accessible right? As long as it is accessible that would be okay @fakhrip

**Review by IritaSee — CHANGES_REQUESTED** [review]
Looks good to me

**Review by IritaSee — APPROVED** [review]
Looking good, sorry last one was a mistake

**Review by IritaSee — APPROVED** [review]

**Review by fakhrip — APPROVED** [review]

**Pull request merged**